Makefile

```diff
@@ -474,12 +465,14 @@ endif
 	@ecs-cli compose -f docker-compose.prod.yml up
 
 
-
-# Deprecated targets
 deploy:
-	@echo $@ ERROR: see deployment steps for accelerate.
-	@echo see: https://github.com/masschallenge/standards/blob/376d290b41a202acc5c2263d7275ba4a57330ad7/create_new_release.md#deploy-to-staging
+	@if [ "$$IMAGE_TAG" != "development" ] && [ "$$IMAGE_TAG" != "AC-6678" ]; then exit 1; fi;
+	@pip install --upgrade certifi pyopenssl requests[security] ndg-httpsclient pyasn1 pip botocore
+	@curl -s https://raw.githubusercontent.com/silinternational/ecs-deploy/master/ecs-deploy | sudo tee /usr/bin/ecs-deploy
```

Comment:
could we/should we store this script on our own/this repo?

Reply:
We can, I just think in case AWS changes its API for some reason..that overhead would fall on us to update. Pulling it from the remote allows us to always have a "valid" script